test/parser/parserize.ml

```diff
@@ -19,10 +19,10 @@ let rec txt_of_expr = function
     let v1 = txt_of_expr e1 and op1 = txt_of_op op and v2 = txt_of_expr e2 in
     "Binop(" ^ v1 ^ ", " ^ op1 ^ ", " ^ v2 ^ ")"
   | Call(f, args) -> let args1 = List.map txt_of_expr args in
-    "Call(Id(" ^ f ^ "), [" ^ String.concat " ; " args1 ^ "])"
```

Comment:
right now we're parsing functions as `Call of string * expr list`, not with IDs. We might eventually want to parse the function name as an ID and not a string though